xlators/mount/fuse/src/fuse-bridge.c

```diff
@@ -466,16 +468,16 @@ fuse_invalidate_entry(xlator_t *this, uint64_t fuse_ino)
 
         gf_log("glusterfs-fuse", GF_LOG_TRACE,
                "INVALIDATE entry: %" PRIu64 "/%s (gfid:%s)", fnieo->parent,
-               dentry->name, uuid_utoa(inode->gfid));
+               dentry->name, gfid_str);
 
         if (dentry->parent) {
             fuse_log_eh(this, "Invalidated entry %s (parent: %s) gfid:%s",
                         dentry->name, uuid_utoa(dentry->parent->gfid),
```

Comment:
nit: if fnieo->parent == 1, it's because inode_to_fuse_nodeid() determined dentry->parent is root gfid, and we could log this fact instead of uuid_utoa(dentry->parent->gfid)

Not sure it's worth the hassle though.

Reply:
Root's gfid is already special (UUID 1), this allows to distinguish it by eye, I don't think we should handcraft code to make it even more special.